petclinic-frontend/src/features/products/components/Product.tsx

```diff
@@ -51,14 +59,14 @@ function Product({ product }: { product: ProductModel }): JSX.Element {
     }
   };
 
-  const handleProductClick = async (productId: string): Promise<void> => {
-    try {
-      const product = await getProductByProductId(productId);
-      setSelectedProduct(product);
-    } catch (error) {
-      console.error('Failed to fetch product details:', error);
-    }
-  };
+  // const handleProductClick = async (productId: string): Promise<void> => {
+  //   try {
+  //     const product = await getProductByProductId(productId);
+  //     setSelectedProduct(product);
+  //   } catch (error) {
+  //     console.error('Failed to fetch product details:', error);
+  //   }
+  // };
```

Comment:
i think its better to delete this completely as you implemented a new product handle click above